packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -165,4 +165,41 @@ export class IncomingPayment
 
     return data
   }
+
+  $formatJson(json: Pojo): Pojo {
+    json = super.$formatJson(json)
+    return {
+      id: json.id,
+      accountId: json.accountId,
+      incomingAmount: this.incomingAmount
+        ? {
+            ...json.incomingAmount,
+            value: json.incomingAmount.value.toString()
+          }
+        : null,
```

Comment:
As mentioned on the [Rafiki call](https://github.com/interledger/rafiki/discussions/346), we should decide between either excluding optional fields from Open Payments responses when null, or marking them as [`nullable`](https://swagger.io/docs/specification/data-models/data-types/#null) in the OpenAPI spec. I'm leaning towards the former.

Reply:
Me too. I'll exclude them.